scripts/build.py

```diff
@@ -339,6 +339,12 @@ def __del__(self):
             print('ERROR: Pending futures (%d)' % len(self.futures))
             os._exit(5)
 
+    def get_python(self):
+        if 'x86_64-darwin' in self.host2:
+            if 'x86_64-darwin' == self.target_platform:
+                return 'arch -x86_64 python'
+        return 'python'
```

Comment:
With this change it worked with building with Apple's python (which contains arm64/x86_64).

It needs more work, but let's do that when adding support for the M1 build.

Reply:
So why more specifically is this needed? Building the engine on a M1 or did something change with a recent OS update?